backend/README.md

```diff
@@ -4,115 +4,180 @@ Node.js app which subscribes to the blocks on the source chain and sends block d
+      "feedId": 10
+    },
+    {
+      "httpUrl": "https://parallel-heiko.api.onfinality.io/public",
+      "paraId": 2085,
+      "accountSeed": "//Alice//2085",
+      "feedId": 11
+    },
+    {
+      "httpUrl": "https://kintsugi.api.onfinality.io/public",
+      "paraId": 2092,
+      "accountSeed": "//Alice//2092",
+      "feedId": 12
+    }
+  ]
 }
 ```
 
 Where:
-* `targetChainUrl` - WebSocket JSON-RPC endpoint URL of the target (Subspace) chain where transactions with blocks will be sent
-* `downloadedArchivePath` - optional path to downloaded archive of blocks for a particular chain as RocksDB database (can be created with `tools/download-substrate-blocks` script)
-* `httpUrl` - HTTP JSON-RPC endpoint URL of a Substrate-based chain
-* `wsUrl` - WebSocket JSON-RPC endpoint URL of the main Substrate-based chain (in most cases relay chain like Kusama or Polkadot, but can be used with any other chain too)
-* `paraId` - ID of a parachain or parathread under above relay chain
-* `accountSeed` - seed for the account that will be used on target chain for submitting transactions with blocks for particular chain (all such accounts can be funded with `tools/fund-accounts` script)
-* `feedId` - ID of the feed already created on Subspace chain into which archived blocks will go (`tools/create-feeds` script can be used to create feeds for accounts in the config file)
+
+- `targetChainUrl` - WebSocket JSON-RPC endpoint URL of the target (Subspace) chain where transactions with blocks will be sent
+- `downloadedArchivePath` - optional path to downloaded archive of blocks for a particular chain as RocksDB database (can be created with `tools/download-substrate-blocks` script)
+- `httpUrl` - HTTP JSON-RPC endpoint URL of a Substrate-based chain
+- `wsUrl` - WebSocket JSON-RPC endpoint URL of the main Substrate-based chain (in most cases relay chain like Kusama or Polkadot, but can be used with any other chain too)
+- `paraId` - ID of a parachain or parathread under above relay chain
+- `accountSeed` - seed for the account that will be used on target chain for submitting transactions with blocks for particular chain (all such accounts can be funded with `tools/fund-accounts` script)
+- `feedId` - ID of the feed already created on Subspace chain into which archived blocks will go (`tools/create-feeds` script can be used to create feeds for accounts in the config file)
 
 </details>
 
 ## Scripts
+
 - `npm start` - run application
 - `npm run lint` - check codebase with Eslint
 - `npm run build` - build project
 
 License: Apache-2.0
+
+## Docker
+
+Instructions to build and run with docker:
+
+<details>
+### Build
+
+```
+docker build . -t relayer-backend
+```
+
+### Run account funding.
+
+Replace **CONFIG_DIR** with your local path to **config.json**
+
+```
+docker run -it -e  FUNDS_ACCOUNT_SEED="//Alice" -e CHAIN_CONFIG_PATH="/app/config.json" --volume /CONFIG_DIR/config.json:/app/config.json  --network subspace relayer-backend fund-accounts
```

Comment:
This will work, but a bit strange to use `/app` here as the rest of things are in the root in the final image.